platform/nativescript/runtime/components/list-view.js

```diff
@@ -38,6 +52,18 @@ export default {
             this.$emit('itemTap', args)
         },
 
+        onLoaded(args) {
+            this.$emit('loaded', args)
+        },
+
+        onUnloaded(args) {
+            this.$emit('unloaded', args)
+        },
+
+        onLoadMoreItems(args) {
+            this.$emit('loadMoreItems', args)
+        },
+
         setupTemplates() {
             const self = this
```

Comment:
Hmm, it's used in the `createView()` fn, but since it's inside arrow functions it might be redundant, will check that later, not part of this PR anyways!